openpype/hosts/fusion/deploy/MenuScripts/install_pip_package.py

```diff
@@ -0,0 +1,56 @@
+import os
+import platform
+import subprocess
+from pathlib import Path
+
+
+def get_python_home() -> Path:
+    python_home = Path(os.__file__).parent.parent
+    return python_home
+
+
+def get_python_executable(python_home):
+    if platform.system() == "Windows":
```

Comment:
Wouldn't `shutil.which("python")` achieve the same?

Reply:
`shutil.which` would likely search `PATH` env var which isn't necessarily the case - also, as can be seen from the code we're actually looking for `python` on Windows and `python3` on other platforms, as such a single `which("python")` would not work, right?

Reply:
I think most platforms do a symlink of `python` -> `python3`; but after testing it does pick up whatever is the first in `PATH` so it's not the same.